### PR TITLE
Tidying up

### PR DESCRIPTION
**Pull request by holygits**
Still some things to to do but this is a start
- Update README
- Move release notes to changelogs
- Correct author field

### CODE REVIEW

File: packages/types/CHANGELOG.md [hunk 1]
```diff
@@ -3,7 +3,50 @@
...
+    - Updated Azalea metadata for runtime version 36 (#122)
+
+## [1.0.3](https://github.com/cennznet/api.js/compare/release/1.0.2...release/1.0.3) (13/07/2020)
+
+    Adds:
+    - Support CENNZX liquidity RPCs (#117)
+    - Slim metadata generation script (#120)
+    - Support on-chain asset registry (#116)
+    Fixes:
+    - Corrected some generated example snippets (#121)
```
Comment (KarishmaBothara) [inline packages/types/CHANGELOG.md:20]: I think, this was after the release.. In the UNRELEASED section.. (corrected some generated example snippets)

File: packages/api/CHANGELOG.md [hunk 2]
```diff
@@ -3,6 +3,27 @@
...
+
+## [1.0.2](https://github.com/cennznet/api.js/compare/release/1.0.1...release/1.0.2) (15/05/2020)
+
+**Note:** Version bump only
+
+## [1.0.1](https://github.com/cennznet/api.js/compare/release/1.0.0...release/1.0.1) (27/03/2020)
+
+**Note:** Version bump only
+
+## [1.0.0](https://github.com/cennznet/api.js/compare/v0.20.5...release/1.0.0) (27/03/2020)
```
Comment (KarishmaBothara) [inline packages/api/CHANGELOG.md:20]: changes log looks great to me.. just curious to investigate if the existing CHANGELOG was auto generated earlier with 
`RUN npx lerna version ${RELEASE_SCOPE} --yes --conventional-commits --force-publish='*' --changelog-preset @cennznet/conventional-changelog-preset`

Comment (holygits) [reply packages/api/CHANGELOG.md:20]: would be much easier than this if it is.
also I don't think we should bump packages if they don't have changes anymore

**Comment by alexsednz** [comment]
LRGTM :)

**Comment by holygits** [comment]
test failure seems unrelated to this PR

**Pull request merged**